app/models/aggregation.rb

```diff
@@ -14,8 +14,11 @@ class Aggregation < ActiveRecord::Base
   validate :aggregation, :workflow_version_present
 
   def self.scope_for(action, user, opts={})
-    return super unless (action == :show || action == :index) && !user.is_admin?
-    joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
+    if (action == :show || action == :index) && !user.is_admin?
+      joins(:workflow).merge(Workflow.scope_for(:update, user, opts))
```

Comment:
Had a look at using an `or` query but can't keep getting malformed queries. Any chance you can take a look? latest in https://github.com/camallen/Panoptes/commits/public_aggregations